lifecycle-operator/apis/lifecycle/v1alpha3/keptntaskdefinition_types.go

```diff
@@ -64,6 +64,8 @@ type KeptnTaskDefinitionSpec struct {
 	// The timer starts when the status shows up to be Complete or Failed.
 	// +kubebuilder:default:=300
 	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty"`
+	// ImagePullSecrets is an optional field to specify the names of secrets to use for pulling container images
+	ImagePullSecrets []v1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
```

Comment:
If it's optional, let's mark it :) 
```suggestion
	// ImagePullSecrets is an optional field to specify the names of secrets to use for pulling container images
	// +optional
	ImagePullSecrets []v1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
```